### ISSUE TITLE
Rebalance Smithing

### ISSUE THREAD

**Issue by Shadow-Quill** [ARF-SS13/coyote-bayou#482]
Right no, smithing can make OP melee weapons, and is in general not very true to fallout. It needs to be nerfed and reflavored

**Comment by Marrone321** [comment]
You need to organize your thoughts, here are some steps you can take so we can tackle this:

Cohesively identify which smithable items are overpowered, which ones are not, explain how they are overpowered in contrast to things you can obtain, and keep in mind the time and resource investment you would need to achieve the results.

From there we can begin tweaking things to balance them out, but there should be a way for players to craft high-level melee weapons and forging is definitely the most advanced way to do it - its not like you just need tools/materials at a work bench, you need fuel, a furnace, anvil, blacksmithing gloves, a hammer, an ignot and many other things, it is not easy and simple to make but that is why you can make some interesting and powerful wewapons.

I do ask that if you are going to report an issue, especially on balance and not a bug, you should consider being as **specific** as possible with a game plan for a solution that is flexible enough to be altered once you recieve feedback.

Looking forward to your prepatory work before we get anything done on this, obviously if someone else wants to do that feel free to post it here for those who care about smithing so we can get some feedback.

**Comment by Shadow-Quill** [comment]
> You need to organize your thoughts, here are some steps you can take so we can tackle this:

These have been taken from the task list project board, I have not written any of those, I've only created them.
For this particular one, @Superlagg was the creator.

**Comment by Brimcon** [comment]
I disagree that smithing weapons are OP. They are weaker than other more readily available weapons even at the highest quality craft you can make (8). Even if you were to make an Adamantine weapon (Pre-War Ingot which is rare enough on its own) You can just go find a Mob Raider, kill them for their fire axe or baseball bat, and you have a better weapon than the woodchopper or mace that you can craft with Titanium. A Bone spear out classes a smithed spear by 3 points of force, no matter what material or quality used in the smithing process.

As someone who has been primarily smithing the past few days, my efforts feel rather useless and a waste of Titanium, Metal, and welder fuel. At most they are a tool to spark roleplay or other things.
